### PR TITLE
remove stale pricing pages

### PR DESCRIPTION
**Pull request by tvald**
Keeping pricing pages in sync was challenging. 

The docs pricing pages had various outdated statements, including mention of $0.50/gb pricing on Topics.

Removing the pages so that we don't have to worry about them any more.

Updated links and redirects as well, so this page depends on the pending redirect PR.

### CODE REVIEW

File: i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Momento Cache はこの古典的なサーバーレスの定義に完璧に適合
 
 第二に、Momento Cache はキャッシュ管理に関する設定を抽象化しています。キャッシュに必要なインスタンスの数を指定することも、最大メモリサイズを考えることも必要ありません。Momento はクラウドに特化して作られていて、現代的なクラウド基盤の弾力性やスケーラビリティを最大活用しています。皆さんはキャッシュに必要なデータをどれだけでも保存することができて、Momento がそれをシームレスに取り扱います。
 
-最後に、Momento Cache は従量課金のモデルです。上述の様に、利用の有無に関係なく、事前に特定のキャッシュの種類やクラスターサイズを選択することはありません。Momento は Momento Cache と Momento Topics で行われたデータ転送の出入りに対して[課金します](../manage/pricing)。それ以外は全て含まれています。これによって、料金は皆さんの支配下にあって、アプリケーションの変更が直接的に請求金額に影響を与えます。
+最後に、Momento Cache は従量課金のモデルです。上述の様に、利用の有無に関係なく、事前に特定のキャッシュの種類やクラスターサイズを選択することはありません。Momento は Momento Cache と Momento Topics で行われたデータ転送の出入りに対して[課金します](https://www.gomomento.com/pricing)。それ以外は全て含まれています。これによって、料金は皆さんの支配下にあって、アプリケーションの変更が直接的に請求金額に影響を与えます。
```
Comment (malandis) [inline i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md]: FYI the English pricing page is as above but the path is different on the Japanese page (`pricing` vs `price`):

https://www.gomomento.com/pricing

vs

https://jp.gomomento.com/price/

So when I follow the link above to the English page and select Japanese language, I am redirected to the Japanese landing page https://jp.gomomento.com, which I doubt we intend :)

File: i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ description: リアルタイムなアプリケーション通信のための、
 
 # Momento Topics とは？
 
-Momento Topics は、最高に洗練されたサーバーレスの pub/sub (パブリッシュ/サブスクライブ) メッセージングパターンで、分散システム間のリアルタイム通信を可能にします。アプリのパブリッシャーから送信されたアイテムを受信するために、アプリが *topics*、つまり専用のチャネルでサブスクライブすることを可能にします。Momento Topics サービスはサーバーレスであるため、インフラストラクチャを作成したり管理したりする必要はなく、pub/subするトピックを作成する必要もありません。あるトピックにアイテムをパブリッシュすると、そのアイテムがまだ存在しない場合はトピックが自動的に作成されます。トピックはMomento Cacheを基盤として利用するため、スケーリング設定も不要です。サービスの[転送されたデータの入出力](./manage/pricing.md)に対してのみ課金されます。
+Momento Topics は、最高に洗練されたサーバーレスの pub/sub (パブリッシュ/サブスクライブ) メッセージングパターンで、分散システム間のリアルタイム通信を可能にします。アプリのパブリッシャーから送信されたアイテムを受信するために、アプリが *topics*、つまり専用のチャネルでサブスクライブすることを可能にします。Momento Topics サービスはサーバーレスであるため、インフラストラクチャを作成したり管理したりする必要はなく、pub/subするトピックを作成する必要もありません。あるトピックにアイテムをパブリッシュすると、そのアイテムがまだ存在しない場合はトピックが自動的に作成されます。トピックはMomento Cacheを基盤として利用するため、スケーリング設定も不要です。
```
Comment (malandis) [inline i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md:11]: Did you intend to completely remove the link here as opposed to replacing?

**Review by malandis — COMMENTED** [review]
👍 from me after considering a few small issues:

File: i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md (hunk 2 above)
Comment (tvald) [reply i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md:11]: Yes, this sentence was also removed from the English version.

File: i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md (hunk 1 above)
Comment (tvald) [reply i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md]: Good catch, will fix.

**Comment by malandis** [comment]
Will approve once rebased/merged with the build fix

**Review by malandis — APPROVED** [review]

**Pull request merged**